common/server.conf

```diff
@@ -1,6 +1,3 @@
-ssl_certificate common/snakeoil.crt;
```

Comment:
We are not removing it, it just got moved into a separate file and DC/OS package `build` script will have to be adjusted accodingly. Check the Dockerfile changes.